src/components/filter/GenresTab.js

```diff
@@ -0,0 +1,21 @@
+import {GENRES} from "../../constants/Constant";
+import {Col, Tabs} from "antd";
+import React from 'react';
+import classes from "./styles.module.css";
```

Comment:
Стили лучше именовать как и название компонента, потом проще с ними будет работать и меньше колизий в названиях. 
```suggestion
import classes from './GenresTab.module.css';
```
И далее по коду те же рекомендации.

Reply:
Renamed